static/app/views/starfish/views/spans/spanSummaryPanel/useSpanSeenEvent.tsx

```diff
@@ -0,0 +1,67 @@
+import {useQuery} from 'sentry/utils/queryClient';
+import {useQueryGetEvent} from 'sentry/views/starfish/modules/databaseModule/queries';
+import {HOST} from 'sentry/views/starfish/utils/constants';
+import type {Span} from 'sentry/views/starfish/views/spans/spanSummaryPanel/types';
+
+type HasTransaction = {
+  transaction_id: string;
+};
+
+function useFirstSeenSpan(span?: Span, referrer = 'first-seen-span') {
+  const query = span
+    ? `
+    SELECT transaction_id
+    FROM spans_experimental_starfish
+    WHERE group_id = '${span.group_id}'
+    ORDER BY start_timestamp ASC
+    LIMIT 1
+ `
+    : null;
+
+  const result = useQuery<(Span & HasTransaction)[]>({
+    queryKey: ['span', 'first', span?.group_id],
+    queryFn: () =>
+      fetch(`${HOST}/?query=${query}&referrer=${referrer}`).then(res => res.json()),
```

Comment:
Thoughts on something like this
```suggestion
      fetch(`${HOST}/?query=${query}&referrer=${referrer}-first-seen-span`).then(res => res.json()),
```
And referer we would pass in `event-panel`.
That way you'll always know where the query is defined and where it's being called.